Week1/francisco-torres/unit-test/src/test/java/com/globant/bootcamp/spring/unittest/kata/AccountServiceImplTest.java

```
@@ -0,0 +1,85 @@
+package com.globant.bootcamp.spring.unittest.kata;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import springbootcamp.unittest.kata.business.AccountServiceImpl;
+import springbootcamp.unittest.kata.data.AccountRepository;
+import springbootcamp.unittest.kata.domain.Account;
+import springbootcamp.unittest.kata.domain.AccountType;
+import java.lang.reflect.Constructor;
+import static org.mockito.Mockito.when;
+
+public class AccountServiceImplTest {
+
+    @Mock
+    private AccountRepository accountRepository;
+
+    @InjectMocks
+    private AccountServiceImpl accountService;
+
+    private Account currentAccount;
+    private Account savingAccount;
+    private int initialBalanceCurrentAccount;
+    private int initialBalanceSavingAccount;
+    private int transferAmount;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.initMocks(this);
+
+        try {
+            Constructor<Account> c = Account.class.getDeclaredConstructor(String.class, int.class, AccountType.class);
+            c.setAccessible(true);
+            currentAccount = c.newInstance("Ana", 0, AccountType.CURRENT);
+            savingAccount = c.newInstance("Pedro", 0, AccountType.SAVING);
+        } catch (Exception e) {
```

In unit tests we should catch exceptions, in this case, let it throw. I